src/components/WeekDayPicker/__test__/weekDayPicker.spec.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { useUniqueIdentifier } from '../../../libs/hooks';
+import WeekDayPicker from '../';
+import StyledTextError from '../../Input/styled/errorText';
+
+jest.mock('../../../libs/hooks/useUniqueIdentifier', () => jest.fn(() => 'week-day'));
+
+describe('<WeekDayPicker />', () => {
+    it('should render a label when label prop is passed', () => {
+        const component = mount(<WeekDayPicker label="My label" />);
+        expect(component.find('legend').length).toBe(1);
+    });
+    it('should render bottom help text when bottomHelpText prop is passed', () => {
+        const component = mount(<WeekDayPicker bottomHelpText="My help text" />);
+        expect(component.find('div[children="My help text"]').length).toBe(1);
```

Comment:
use StyledHelpText since div is an implementation detail of the HelpText component